Library/Formula/python.rb

```diff
@@ -106,6 +106,17 @@ def install
       end
       args << cflags
       args << ldflags
+    elsif OS.linux?
+      hints = ["openssl"]
+      hints << "readline" if build.with? "readline"
+      hints << "sqlite" if build.with? "sqlite"
+      hints << "gdbm" if build.with? "gdbm"
+      hints << "tcl-tk" if build.with? "tcl-tk"
+      hint_includes = hints.map { |f| "-I#{Formula[f].opt_include}" }
+      hint_libs = hints.map { |f| "-L#{Formula[f].opt_lib}" }
+      cppflags = "CPPFLAGS=#{hint_includes.join " "}"
+      ldflags = "LDFLAGS=#{hint_libs.join " "}"
+      args << cppflags << ldflags
```

Comment:
It may; I hadn't appreciated that these formulas aren't keg-only on Linux. I'll try that later.